core/src/main/scala/nl/vroste/zio/kinesis/client/producer/ProducerLive.scala

```diff
@@ -54,7 +57,9 @@ private[client] final class ProducerLive[R, R1, T](
         { case (shardId @ _, requests) =>
           ZStream.scoped(ShardMap.md5.orDie).flatMap { digest =>
             if (aggregate)
```

Comment:
Do you really need to execute `ShardMap.md5` here for both cases (when `aggregate` is `true` and when it's `false`)? `digest` is used only in one case...

Reply:
updated